app/assets/javascripts/initializers/initScrolling.js

```diff
@@ -1,26 +1,45 @@
 /* global insertAfter, insertArticles, buildArticleHTML, nextPage:writable, fetching:writable, done:writable, InstantClick */
 
-var client;
+// The purpose of initScrolling is to start paginating the page once you reach the bottom of the page.
```

Comment:
`initScrolling` has a lot of potential to be refactored. For now, I've added some JSdocs to help make the purpose of the functions clearer, however, I'm hoping that we can [refactor it out as part of the migrations to the pack file](https://github.com/forem/forem/issues/19709#issuecomment-1657276012). I'll add some notes on what I think could benefit from refactoring.

Reply:
I agree with potentially refactoring it out entirely; I have noticed it sometimes being executed in strange contexts (like on the notifications page?) probably due to InstantClick weirdness. Thank you for the documentation in the meantime!